library/L1_Peripheral/lpc40xx/watchdog_interface.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+#include <iterator>
+
+namespace sjsu
+{
+
+class WatchdogInterface
+{
+ public:
+     virtual void Initialize(uint32_t timer_constant, uint32_t timer_window,
+     uint32_t timer_warning) = 0;

```

Comment:
```suggestion
     uint32_t timer_warning) const = 0;
```
All L1 peripheral interface methods should be made const.